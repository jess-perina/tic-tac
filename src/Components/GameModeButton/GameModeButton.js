import React, { Component } from 'react'
import M from 'materialize-css';

export default class GameModeButton extends Component {
  componentDidMount() {
    let dropdown = document.querySelectorAll('.dropdown-trigger')
    let options = {
      inDuration: 300,
        outDuration: 300,
        hover: true, // Activate on hover
        coverTrigger: false
    }
    M.Dropdown.init(dropdown, options)
    // M.AutoInit()
  }

  render() {
    return (
      <div>
        <span className="card-title">Change Game Mode</span>
        <a className="dropdown-trigger btn" href="#!" data-target="practice-toggle">{this.props.practiceMode}</a>
        <ul id="practice-toggle" className="dropdown-content">
          <li onClick={this.props.setPractice}>
            <a href="#!">Practice</a>
          </li>
          <li onClick={this.props.setSerious}>
            <a href="#!">Serious</a>
          </li>
        </ul>
      </div>
    )
  }
}
