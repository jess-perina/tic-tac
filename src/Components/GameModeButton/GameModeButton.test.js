import React from 'react'
import { shallow } from 'enzyme'
import GameModeButton from './GameModeButton'

describe('GameModeButton Component', () => {
  it('renders as espected', () => {
    expect(shallow(<GameModeButton />)).toMatchSnapshot()
  })
})