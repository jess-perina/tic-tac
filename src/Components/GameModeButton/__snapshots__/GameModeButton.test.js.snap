// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`GameModeButton Component renders as espected 1`] = `ShallowWrapper {}`;
